src/feature/People/PeopleList/List/styled.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+    max-width: 1368px;
+`;
+
+export const PeopleList = styled.ul`
+    padding: 0;
+    display: flex;
+    flex-direction: row;
+    gap: 24px;
+    flex-wrap: wrap;
+`;
+
+export const Item = styled.li`
+    list-style: none;
+`;
+
+export const Tile = styled.div`
+    background-color: ${({ theme }) => theme.colors.white};
+    padding: 16px;
+    width: 208px;
+    height: 339px;
+    box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+    border-radius: 5px;
+`;
+
+export const Poster = styled.img`
```

Comment:
I need your help here - on Figma there is Poster hidden under it's cover. Hard to say... I mean part of the poster is under the white cover. I used z-index -1, but didn't help. Any idea?

Reply:
This is an important question, I will also have the same problem when the film does not have a poster.
We will try to find a solution together